package.json

```diff
@@ -23,32 +23,31 @@
   "angular-cli": {},
   "scripts": {
     "start": "prx-ng-serve dev",
-    "lint": "tslint \"src/**/*.ts\"",
+    "lint": "tslint \"src/**/*.ts\" -e \"src/testing/components.ts\"",
     "test": "ng test",
     "e2e": "ng e2e",
     "testonce": "ng test --watch=false --browsers=PhantomJS",
     "ci": "ng test --watch=false --browsers=PhantomJS --code-coverage && codecov",
-    "build": "ng build --prod --aot --vc=true",
+    "build": "ng build --prod --aot",
     "serve": "prx-ng-serve dist"
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "^5.1.2",
-    "@angular/compiler": "^5.1.2",
-    "@angular/core": "^5.1.2",
-    "@angular/forms": "^5.1.2",
-    "@angular/platform-browser": "^5.1.2",
-    "@angular/platform-browser-dynamic": "^5.1.2",
-    "@angular/router": "^5.1.2",
-    "angulartics2": "^5.1.2",
+    "@angular/common": "^6.1.10",
+    "@angular/compiler": "^6.1.10",
+    "@angular/core": "^6.1.10",
+    "@angular/forms": "^6.1.10",
+    "@angular/platform-browser": "^6.1.10",
+    "@angular/platform-browser-dynamic": "^6.1.10",
+    "@angular/router": "^6.1.10",
+    "angulartics2": "^6.1.10",
     "codecov": "PRX/codecov-node",
     "core-js": "^2.4.1",
     "evaporate": "^2.1.4",
-    "intl": "^1.2.5",
     "langmap": "0.0.14",
     "moment": "^2.18.1",
     "ng2-dragula": "^1.3.1",
-    "ngx-prx-styleguide": "1.1.1",
+    "ngx-prx-styleguide": "^2.0.0",
```

Comment:
We've actually been pinning this to a specific version, which is currently "2.0.1"

Reply:
Fixed